src/gui/addnewtorrentdialog.h

```diff
@@ -65,6 +66,7 @@ class AddNewTorrentDialog: public QDialog
     static bool isTopLevel();
     static void setTopLevel(bool value);
 
+    static void show(QString source, BitTorrent::AddTorrentParams &inParams, QWidget *parent);
```

Comment:
I suggested this to preserve signature similarity.
Then remove it from next `show()` too.